config.js

```diff
@@ -7,35 +7,35 @@ function getConfig(env) {
     case 'mainnet':
         return {
             networkId: 'mainnet',
-            nodeUrl: 'https://rpc.mainnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.mainnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.mainnet.nearprotocol.com',
-            helperUrl: 'https://helper.mainnet.nearprotocol.com',
+            walletUrl: 'https://wallet.near.org',
+            helperUrl: 'https://helper.mainnet.near.org',
```

Comment:
`https://helper.mainnet.near.org/` not working

Reply:
Neither did work https://helper.mainnet.nearprotocol.com, so I have no idea how to setup DNS for helper.mainnet.near.org. /cc @vgrichina

Reply:
This one works: https://helper.nearprotocol.com/

Reply:
(And yes, I think that means the `production.helperUrl` in this file has been incorrect for a long time)

Reply:
That is testnet, as far as I can tell

Reply:
Since we don't have mainnet Wallet, should we scratch this config completely?

Reply:
I think it makes sense to add it in anticipation of what the domains will be, if we have a high level of certainty that they won't change

Reply:
I don't think it makes sense to create any DNS records for them anyway, so I mark this as resolved.